src/plugins/VectorizationTuner.cpp

```diff
@@ -42,12 +51,12 @@ VectorizationTuner::~VectorizationTuner() {
 void VectorizationTuner::readXML(XMLfileUnits& xmlconfig) {
 	std::string mode = "file";
 	xmlconfig.getNodeValue("mode", mode);
-	if (mode.compare("statistics") == 0) {
-		vtWriter = std::unique_ptr<VTWriterI>(new VTWriterStatistics());
-	} else if (mode.compare("file") == 0) {
-		vtWriter = std::unique_ptr<VTWriterI>(new VTWriter());
+	if (mode == "statistics") {
+		vtWriter.reset(new VTWriterStatistics());
+	} else if (mode == "file") {
+		vtWriter.reset(new VTWriter());
 	} else {
-		global_log->error() << "Unknown FlopRateOutputPlugin::mode. Choose \"stdout\", \"file\" or \"both\"." << endl;
+		global_log->error() << R"(Unknown FlopRateOutputPlugin::mode. Choose "stdout", "file" or "both".)" << endl;
```

Comment:
```suggestion
		global_log->error() << "(Unknown FlopRateOutputPlugin::mode. Choose "stdout", "file" or "both".)" << endl;
```

Reply:
nope no typo. This is a raw string literal.
See: 6) in https://en.cppreference.com/w/cpp/language/string_literal
 or https://www.geeksforgeeks.org/raw-string-literal-c/

Reply:
clang-tidy suggested it :)

Reply:
ok TIL I guess